app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> repr:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> float | Distance:
```

Comment:
Only the `Distance` class instance is returned in this method, make changes in your code where applicable to encounter that.